test/tests.js

```diff
@@ -1 +1,30 @@
-import './example.test.js';
```

Comment:
These tests should be written in the example-tests file and imported into the test.js file.